SCClassLibrary/Common/GUI/PlusGUI/Core/HistoryGui.sc

```diff
@@ -328,18 +301,25 @@ HistoryGui : JITGui {
 	}
 
 	postInlined { |index|
+		thisMethod.deprecated(
+			HistoryGui.methods.detect { |m| m.name == \showLineAt }
+		);
+		this.showLineAt(index);
+	}
```

Comment:
please move this method to the deprecated folder

Reply:
done.